index.html

```diff
@@ -22,17 +22,146 @@
+                <li><img src="images/instagram.png" alt="instagram" class="img1"></li>
+                <li><img src="images/hand.png" alt="hand" class="img1"></li>
+                <li><img src="images/twitter.png" alt="twitter" class="img1"></li>
+                <li><img src="images/eme.png" alt="eme" class="img1"></li>
             </ul>
         </div>
+
+        <div class="second-container">
+            <h2 id="recent">My Recent Works</h2>
+        </div>
+
+        <div class="third-container">
+            <img src="images/placeholder.png" alt="placeholder" class="img2">
+            <h3 id="multi">Multi-Post Sories</h3>
+            <p id="par2">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <nav class="langs">
+                <button class="langbuttons">css</button>
+                <button class="langbuttons">html</button>
+                <button class="langbuttons">bootstrap</button>
+                <button class="langbuttons">Ruby</button>
+            </nav>
+            <button class="project">See Project</button>
+        </div>
+
+        <div class="fourth-container">
+            <h3 class="profesional">Profesional Art Printing Data</h3>
+            <p class="par3">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+            <nav class="langs2">
+                <button class="langbuttons2">html</button>
+                <button class="langbuttons2">bootstrap</button>
+                <button class="langbuttons2">Ruby</button>
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
+
+        <div class="fourth-container">
+            <h3 class="profesional">Profesional Art Printing Data</h3>
+            <p class="par3">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+            <nav class="langs2">
+                <button class="langbuttons2">html</button>
+                <button class="langbuttons2">bootstrap</button>
+                <button class="langbuttons2">Ruby</button>
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
+
+        <div class="fourth-container">
+            <h3 class="profesional">Profesional Art Printing Data</h3>
+            <p class="par3">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+            <nav class="langs2">
+                <button class="langbuttons2">html</button>
+                <button class="langbuttons2">bootstrap</button>
+                <button class="langbuttons2">Ruby</button>
+            </nav>
+            <button class="project2">See Project</button>
+        </div>
```

Comment:
Same this applies to all your `fourth-container` elements:

Kindly use a list `ul` and each of the project technologies (e.g., "JavasScript", "HTML", and "CSS")  should be a `li` element because it is semantic that way. The `nav` element should only be used for the `navbar` menu items since they are used for `navigation`, hence `nav`.

Reply:
I implemented ul and li where requested, but I have a little problem. The elements tend to go more to the right. I think it is because of the bullet point. Please if you can help me to correct this I will be grateful.

Reply:
same here, you can use a `padding: 0;` on the parent `ul` tag to remove the default indentation of `li` items.